cmd/spire-agent/cli/run/run.go

```diff
@@ -229,6 +234,65 @@ func mergeInput(fileInput *Config, cliInput *agentConfig) (*Config, error) {
 	return c, nil
 }
 
+func downloadTrustBundle(ac *agent.Config, c *Config) ([]*x509.Certificate, error) {
+	u, err := url.Parse(c.Agent.TrustBundleURL)
+	if err != nil {
+		return nil, fmt.Errorf("could not parse trust bundle URL: %s", u)
+	}
+
+	if strings.ToLower(u.Scheme) != "https" {
```

Comment:
is not url.Scheme always lowercase? (https://github.com/golang/go/blob/master/src/net/url/url.go#L531)

Reply:
You're right, changed